api/src/test/java/jakarta/data/displaynamegeneration/ReplaceCamelCaseAndUnderscore.java

```diff
@@ -0,0 +1,88 @@
+package jakarta.data.displaynamegeneration;
+
+import org.junit.jupiter.api.DisplayNameGenerator;
+
+import java.lang.reflect.Method;
+
+public class ReplaceCamelCaseAndUnderscore extends DisplayNameGenerator.Standard {
+
+    public ReplaceCamelCaseAndUnderscore() {
+        super();
+    }
+
+    @Override
+    public String generateDisplayNameForClass(Class<?> testClass) {
+        return replaceCamelCaseAndUnderscore(super.generateDisplayNameForClass(testClass));
+    }
+
+    @Override
+    public String generateDisplayNameForNestedClass(Class<?> nestedClass) {
+        return replaceCamelCaseAndUnderscore(super.generateDisplayNameForNestedClass(nestedClass));
+    }
```

Comment:
I think we should only replace camel case for methods not classes.

Reply:
Done